angular-ngrx-scss/src/app/organization/services/organization.interfaces.ts

```diff
@@ -0,0 +1,31 @@
+export interface OrganizationReposApiParams {
+  type?:
+    | 'all'
```

Comment:
question: Is this first | needed, before all? It may not matter, but sort and direction don't have a starting pipe. 🤔

Reply:
I just confirmed prettier enforces this.

Reply:
Interesting! Not a big deal at all, but curious that multi-line ones have a pipe at the start. TIL! lol